packages/tooltip/src/tooltip.ts

```diff
@@ -31,7 +38,13 @@ export class Tooltip extends LitElement {
     }
 
     @property({ reflect: true })
-    public tip: 'top' | 'bottom' | 'left' | 'right' = 'top';
+    public placement: Placement = 'top';
+
+    @property({ attribute: 'data-popper-placement' })
+    public dataPopperPlacement?: Placement;
+
+    @property({ attribute: 'computed-placement', reflect: true })
+    public computedPlacement: SpectrumPlacement = 'top';
```

Comment:
Is it possible to simplify the usage of `placement` based attributed in the style APIs?

Most importantly, I don't like the idea of surfacing a `computed-placement` attribute, if we need all three is there better naming there? Also, should we do the work to make `computed-placement` or it's replacement naming actually `readonly`? Maybe `readonly` is overkill?